benchmarks/cloudsuite3/graph-analytics/run.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+
+set -e
```

Comment:
This is not a good idea. If any command returns an error in the script, the script exits, leaving us with no way to recover or clean up properly. You should remove this new addition (`set -e`) from all scripts. We can add error detection and handling.